From 46034cd505038c4e016f8b34f1fa411471be9e4b Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:07:52 -0400
Subject: [usb] Nokia 5300 should be treated as unusual dev

Message-id: <1410887463-20674-401-git-send-email-dzickus@redhat.com>
Patchwork-id: 95338
O-Subject: [RHEL7 PATCH 400/591] USB: Nokia 5300 should be treated as unusual dev
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 6ed07d45d09bc2aa60e27b845543db9972e22a38
Author: Daniele Forsi <dforsi@gmail.com>
Date:   Mon Apr 28 17:09:11 2014 +0200

    USB: Nokia 5300 should be treated as unusual dev

    Signed-off-by: Daniele Forsi <dforsi@gmail.com>
    Cc: stable <stable@vger.kernel.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/storage/unusual_devs.h b/drivers/usb/storage/unusual_devs.h
index ef09cf5..375fa7c 100644
--- a/drivers/usb/storage/unusual_devs.h
+++ b/drivers/usb/storage/unusual_devs.h
@@ -234,6 +234,13 @@ UNUSUAL_DEV(  0x0421, 0x0495, 0x0370, 0x0370,
   USB_SC_DEVICE, USB_PR_DEVICE, NULL,
   US_FL_MAX_SECTORS_64 ),
 
+/* Reported by Daniele Forsi <dforsi@gmail.com> */
+UNUSUAL_DEV(  0x0421, 0x04b9, 0x0350, 0x0350,
+  "Nokia",
+  "5300",
+  USB_SC_DEVICE, USB_PR_DEVICE, NULL,
+  US_FL_MAX_SECTORS_64 ),
+
 /* Patch submitted by Victor A. Santos <victoraur.santos@gmail.com> */
 UNUSUAL_DEV(  0x0421, 0x05af, 0x0742, 0x0742,
   "Nokia",
-- 
1.7.1